style.css

```diff
@@ -527,51 +560,70 @@ form span {
 /* Mobile Responsiveness */
 @media (max-width: 768px) {
   /* Navigation Section */
-  .menu #menuBtn {
+  header {
+    padding: 0 30px;
+  }
+
+  .hamburger {
     display: block;
+    position: fixed;
+    right: 30px;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+    background-color: #fff;
   }
 
-  .menu ul {
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: #fff;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: #fff;
+  }
+
+  .nav-menu {
     position: fixed;
-    width: 250px;
-    height: calc(100vh - 70px);
-    top: 70px;
-    right: -250px;
-    background-color: #6070ff;
-    border-radius: 30% 0% 0% 80%;
-    display: flex;
-    align-items: center;
+    left: -100%;
+    top: 0;
+    gap: 0;
     flex-direction: column;
-    border-top: 1px solid black;
-    transition: right 0.4s cubic-bezier(1, 0.2, 0.2, 1);
+    background: #6070ff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(10px);
+    height: 100vh;
+    width: 100%;
+    text-align: center;
+    transition: 0.3s;
+    align-items: flex-start;
+    padding: 50px 30px;
+  }
```

Comment:
- [ ] **Good job for using ```backdrop-filter``` :+1: :+1:**  however as requested by the previous reviewer. the background doesn't match that of Figma design :fearful:
 I think it would be better to you remove ```mix-blend-mode: multiply;``` from your codes as the element is multiplied by the background and replaces the background color,
try also to add **opacity to the background-color**  to replicate the Figma design :joy: 
 
Example
```
background: rgba(96, 112, 255, 0.8);
``` 
You may kindly follow this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) for more info on backdrop-filter :joy: